tensorflow/tools/dockerfiles/dockerfiles/cpu-jupyter.Dockerfile

```diff
@@ -58,7 +58,8 @@ RUN ${PIP} install ${TF_PACKAGE}${TF_PACKAGE_VERSION:+==${TF_PACKAGE_VERSION}}
 COPY bashrc /etc/bash.bashrc
 RUN chmod a+rwx /etc/bash.bashrc
 
-RUN ${PIP} install jupyter matplotlib
+# https://github.com/ipython/ipykernel/issues/422
+RUN ${PIP} install jupyter ipykernel==5.1.1 matplotlib
```

Comment:
Pin nbconvert to 4.4.0 too.

That also causes errors.

Reply:
Okay, fixed.